src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/jpa/repository/NonAssociationsRepositoryTest.kt

```diff
@@ -72,7 +72,7 @@ class NonAssociationsRepositoryTest : TestBase() {
 
     val closedBy = "Aldo"
     val closedReason = "They're friends now"
-    val closedAt = LocalDateTime.now()
+    val closedAt = LocalDateTime.now(clock)
```

Comment:
i started feeding in `clock` into a bunch of places in tests but then realised that `whenCreated` comes from spring wizardry so will actually be the real world "now". it would be nice if we can teach *spring* to use a fixed clock during tests but i didn't dig deep enough